### DISCUSSION TITLE
2FA Plasmoid for KDE Plasma 5

### DISCUSSION THREAD

**Discussion by martininsulander** [NHAS/wag#133] [Show and tell]
Hi,
First, I really appreciate the WAG project, and it works very well for me.

Secondly, I created a small plasmoid that shows the status of the VPN, and by clicking on it, you go to the 2FA login page.
So If anyone needs a plasmoid, please check this out:  https://gitlab.com/martin112/wag-2fa-plasmoid
(And I hope it is ok that I borrowed the cute dog)

Best regards from Martin

**Comment by NHAS** [comment]
Howdy!

Glad to hear it's working well! Can't say I've heard of plasmoids before but hey that's awesome always love folk making the ecosystem a bit better. 

I'll have a bit more of an indepth read later on
